ITGA7.py

```diff
@@ -0,0 +1,206 @@
+from __future__ import annotations
+import os
+import sys
+from Bio import SeqIO, SeqUtils, SeqRecord
+from Bio.SeqUtils import gc_fraction
+
+
+class BiologicalSequence:
+    """
+    Represents a generic biological sequence.
+    Attributes:
+    - sequence (str): The biological sequence.
+    Methods:
+    - __len__(): Returns the length of the sequence.
+    - __getitem__(index: int): Returns the character at the specified index.
+    - __str__(): Returns a string representation of the sequence.
+    - is_valid_alphabet(): Checks if the sequence uses a valid alphabet.
+    """
+
+    def __init__(self, sequence: str):
+        self.sequence = sequence
+
+    def __len__(self) -> int:
+        return len(self.sequence)
+
+    def __getitem__(self, index: int) -> str:
+        return self.sequence[index]
+
+    def __str__(self) -> str:
+        return f"{self.sequence}"
```

Comment:
оригинально, но я бы сделал все же 
```suggestion
        return str(self.sequence)
```

Reply:
Но ведь sequence это уже str